Cargo.lock

```diff
@@ -4026,9 +4026,9 @@ checksum = "e6d5a32815ae3f33302d95fdcb2ce17862f8c65363dcfd29360480ba1001fc9c"
 
 [[package]]
 name = "futures"
-version = "0.3.30"
+version = "0.3.29"
```

Comment:
Is there any specific reason future libraries are reverted to earlier version?  If not please undo the changes in `Cargo.lock`